src/System.Device.Gpio/System/Device/I2c/I2cDevice.cs

```diff
@@ -35,7 +35,12 @@ public static I2cDevice Create(I2cConnectionSettings settings)
     /// Reads a byte from the I2C device.
     /// </summary>
     /// <returns>A byte read from the I2C device.</returns>
-    public abstract byte ReadByte();
+    public virtual unsafe byte ReadByte()
```

Comment:
Does this really require "unsafe"? I see no pointers here.

Reply:
Does this apply to the `WriteByte` method as well?

Reply:
Sure, yes.